src/calva-fmt/src/config.ts

```diff
@@ -55,11 +69,7 @@ function readConfiguration() {
     }
 }
 
-export function setLspFormatConfig(config: string) {
-    lspFormatConfig = config;
-}
-
-export function getConfig() {
-    const config = readConfiguration();
+export async function getConfig() {
```

Comment:
This function is actually redundant. You can just export `readConfiguration` and await it where you call it, which is the same as awaiting a call to this function.